aead/src/stream.rs

```diff
@@ -322,12 +323,13 @@ macro_rules! impl_stream_object {
             pub fn $last_in_place_method(
                 self,
                 associated_data: &[u8],
-                buffer: &mut dyn Buffer
+                buffer: &mut dyn Buffer,
             ) -> Result<(), Error> {
-                self.stream.$in_place_op(self.position, true, associated_data, buffer)
+                self.stream
+                    .$in_place_op(self.position, true, associated_data, buffer)
             }
         }
-    }
+    };
 }
 
 impl_stream_object!(
```

Comment:
Huh, did `rustfmt` add new formatting rules for macros or something?

Reply:
Hm, it could be a nightly thing. I will revert the `aead` formatting changes.